cmd/argocd/commands/app.go

```diff
@@ -337,6 +337,7 @@ func NewApplicationGetCommand(clientOpts *argocdclient.ClientOptions) *cobra.Com
 		refresh        bool
 		hardRefresh    bool
 		output         string
+		timeout        int
```

Comment:
```suggestion
		timeout        uiint
```
This should be `uint`.